EcosiaTests/SnapshotTests/EcosiaSnapshotTests.xctestplan

```diff
@@ -15,7 +15,8 @@
     {
       "parallelizable" : true,
       "skippedTests" : [
-        "NTPTests"
+        "NTPTests",
+        "SnapshotBaseTests"
```

Comment:
Why do we need to skip `SnapshotBaseTests`?

Reply:
`SnapshotBaseTests` is the base class from which all tests inherit. No tests are defined there so no need to run it and have it in our test results 👍